src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -48,6 +48,66 @@ const QSet<QString> zeroWidthEmotes{
 
 namespace chatterino {
 
+namespace {
+
+    QString stylizeUsername(const QString &username, const Message &message)
+    {
+        auto app = getApp();
+
+        const QString &localizedName = message.localizedName;
+        bool hasLocalizedName = !localizedName.isEmpty();
+
+        // The full string that will be rendered in the chat widget
+        QString usernameText;
+
+        switch (getSettings()->usernameDisplayMode.getValue())
+        {
+            case UsernameDisplayMode::Username: {
+                usernameText = username;
+            }
+            break;
+
+            case UsernameDisplayMode::LocalizedName: {
+                if (hasLocalizedName)
+                {
+                    usernameText = localizedName;
+                }
+                else
+                {
+                    usernameText = username;
+                }
+            }
+            break;
+
+            default:
+            case UsernameDisplayMode::UsernameAndLocalizedName: {
+                if (hasLocalizedName)
+                {
+                    usernameText = username + "(" + localizedName + ")";
+                }
+                else
+                {
+                    usernameText = username;
+                }
```

Comment:
Can be simplified with a ternary operator.

Reply:
1) The code is just copy-pasted from another function since it needed to be reused, so not super in scope of this PR to fix it
2) I would argue that using ternary operators here wouldn't necessarily make the code more readable